setup.py

```diff
@@ -31,11 +36,40 @@ def include_dirs(self, include_dirs):
         self._include_dirs = include_dirs
 
 
+def get_liberfa_versions(path=os.path.join(LIBERFADIR, 'configure.ac')):
+    assert os.path.exists(LIBERFADIR)
+
+    with open(path) as fd:
+        s = fd.read()
+
+    mobj = re.search(r'AC_INIT\(\[erfa\],\[(?P<version>1.7.1)\]\)', s)
```

Comment:
This regex should not have `1.7.1` in it! (rather `[0-9.]+`?)

Reply:
ops, you are right, I will fix